packages/react/src/menu-item/MenuItem.css.ts

```diff
@@ -0,0 +1,62 @@
+import { theme } from "../styles";
+import { recipe, style } from "../vanilla-extract";
+
+export const item = recipe({
+  base: [
+    {
+      alignItems: "center",
+      display: "flex",
+      flexDirection: "row",
+      fontSize: "md",
+      h: "40",
+      justifyContent: "start",
+      px: "xs",
+      rounded: "sm",
+    },
+    style({
+      selectors: {
+        "&:active:not([data-disabled])": {
+          background: theme.colors["neutral.200"],
```

Comment:
As we are only changing the color.
```suggestion
          backgroundColor: theme.colors["neutral.200"],
```